web/packages/api/src/toEthereumSnowbridgeV2.ts

```diff
@@ -380,10 +429,6 @@ export const estimateFeesFromParachains = async (
             ),
             feePadPercentage,
         )
-        returnToSenderExecutionFeeDOT = padFeeByPercentage(
-            await sourceParachainImpl.calculateXcmFee(deliveryXcm.returnToSenderXcm, DOT_LOCATION),
-            feePadPercentage,
-        )
```

Comment:
Why do we not need these anymore?

Reply:
If I understand correctly, the returnToSender XCM serves as a fallback when an asset becomes trapped on AH. But since we now have dry-run support and setHints(Claimer) enabled in V2, it’s no longer necessary. This also saves a bit on end-user fees.